libs/remix-astwalker/src/astWalker.ts

```diff
@@ -12,7 +12,7 @@ const isObject = function(obj: any): boolean {
 export function isAstNode(node: Record<string, unknown>): boolean {
   return (
     isObject(node) &&
-    'id' in node &&
+    // 'id' in node &&
```

Comment:
the id is removed because it seems `YulBlock` ast nodes don't have ids...

Reply:
yes